Assets/Editor/CustomGui.cs

```diff
@@ -21,6 +21,14 @@ public class CustomGUI : EditorWindow
     public List<int> found; // A list of found trees.
     public List<int> treesToDelete; // A list of trees to delete
     public int layers; // The amount of layers in the Tree.
+    int GUIwidth;
+    int GUIheight;
+   
+    //variables that affect the lines thickness, colour, etc...
+    int shadowLine = 1;
+    int shadowEdge = 4;
```

Comment:
Comment why magic number is 4.

Reply:
shadowEdge is the thickness of the line when it starts and ends, I am looking into ways to have the user selected these numbers instead of them being hard coded,.